src/main/java/core/basesyntax/figure/Figure.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.figure;
+
+import core.basesyntax.behaviour.FigureCreation;
+import core.basesyntax.behaviour.FigureDraw;
+import core.basesyntax.tools.Color;
+import java.util.Random;
+
+public abstract class Figure implements FigureDraw, FigureCreation {

```

Comment:
Don’t use abstract classes to set behaviour for classes (https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#dont-use-abstract-classes-to-set-behaviour-for-classes)
There is not need to store area in Figure